pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -841,8 +840,8 @@ func TestMakeDeployment(t *testing.T) {
 			if err != nil {
 				t.Fatalf("got unexpected error: %v", err)
 			}
-			if diff := cmp.Diff(test.want, got, cmpopts.IgnoreUnexported(resource.Quantity{})); diff != "" {
-				t.Errorf("MakeDeployment (-want, +got) = %v", diff)
+			if diff := cmp.Diff(test.want, got, cmp.AllowUnexported(resource.Quantity{})); diff != "" {
```

Comment:
There's one more of these in `pkg/reconciler/revision/resources/pa_test.go`.

Reply:
Removed altogether. PA has no resource in it.